### PR TITLE
[SDX-791] Added EscapeKeyHandler

### PR DESCRIPTION
**Pull request by jenslonkowski**

### CODE REVIEW

**Comment by jenslonkowski** [comment]
Good catch @ahollesen - I removed `src/services.js`.

File: src/framework/services/event/EscapeKeyHandler.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    this.callback = callback;
+
+    this.onKeyDown = this.onKeyDown.bind(this);
+    this.removeListener = this.removeListener.bind(this);
+
+    document.addEventListener('keydown', this.onKeyDown);
+  }
+
+  onKeyDown(event) {
+    if (event.keyCode === 27) {
```
Comment (mohsinhijazee) [inline src/framework/services/event/EscapeKeyHandler.js]: If possible to introduce a constant here something like `ESCAPE_KEY`

File: src/framework/services/event/EscapeKeyHandler.spec.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+
+import EscapeKeyHandler from './EscapeKeyHandler';
+
+describe('EscapeKeyHandler', () => {
+  describe('constructor', () => {
+    it('sets callback of instance to passed method', () => {
+      const callback = () => undefined;
```
Comment (mohsinhijazee) [inline src/framework/services/event/EscapeKeyHandler.spec.js]: Could be `()=>{}`

Comment (jenslonkowski) [inline src/framework/services/event/EscapeKeyHandler.spec.js]: Our code style doesn't allow this (http://eslint.org/docs/rules/no-empty-function).

**Pull request merged**